src/index.html

```diff
@@ -4,9 +4,49 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/styles.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__block">
+        <div class="block block--1"></div>
+      </div>
+      <div class="calendar__block">
+        <div class="block block--2"></div>
+        <div class="block block--3"></div>
+        <div class="block block--4"></div>
+        <div class="block block--5"></div>
+        <div class="block block--6"></div>
+        <div class="block block--7"></div>
+        <div class="block block--8"></div>
+      </div>
+      <div class="calendar__block">
+        <div class="block block--9"></div>
+        <div class="block block--10"></div>
+        <div class="block block--11"></div>
+        <div class="block block--12"></div>
+        <div class="block block--13"></div>
+        <div class="block block--14"></div>
+        <div class="block block--15"></div>
+      </div>
+      <div class="calendar__block">
+        <div class="block block--16"> </div>
+        <div class="block block--17"></div>
+        <div class="block block--18"></div>
+        <div class="block block--19"></div>
+        <div class="block block--20"></div>
+        <div class="block block--21"></div>
+        <div class="block block--22"></div>
+      </div>
+      <div class="calendar__block">
+        <div class="block block--23"></div>
+        <div class="block block--24"></div>
+        <div class="block block--25"></div>
+        <div class="block block--26"></div>
+        <div class="block block--27"></div>
+        <div class="block block--28"></div>
+        <div class="block block--29"></div>
```

Comment:
you don't need modificator  for every blocks, it's supposed be only for calendar blocks parrent